.kokoro/continuous/bigtable-it.cfg

```diff
@@ -8,7 +8,7 @@ env_vars: {
 
 env_vars: {
     key: "INTEGRATION_TEST_ARGS"
-    value: "google-cloud-clients/google-cloud-bigtable -Dbigtable.env=prod -Dbigtable.project=gcloud-devel -Dbigtable.instance=google-cloud-bigtable -Dbigtable.table=integration-tests"
+    value: "google-cloud-clients/google-cloud-bigtable -P bigtable-emulator-it,bigtable-prod-it -Dbigtable.project=gcloud-devel -Dbigtable.instance=google-cloud-bigtable -Dbigtable.table=integration-tests"
```

Comment:
@chingor13 can you just double check the kokoro configs

Reply:
Anything in particular that you are concerned about?